external/fv3fit/tests/training/test_autoencoder.py

```diff
@@ -29,15 +29,35 @@ def get_synthetic_waves_tfdataset(nsamples, nbatch, ntime, nx, nz):
         nz=nz,
         wave_type="sinusoidal",
         scalar_names=["b"],
-        scale_min=0.5,
-        scale_max=1.5,
+        scale_min=1.0,
+        scale_max=1.0,
```

Comment:
With the architecture matching the paper, the NN can't directly store the amplitude of the input waves, and because those amplitudes aren't correlated with anything, there's no way to infer them. I'll keep an eye out for this affecting scientific results, e.g. if amplitudes of generated patterns don't make sense. But it should stand a better chance on non-synthetic data, where amplitudes are correlated with patterns.

Reply:
I added the instance normalization they apply to the first convolutional block's output, which makes it harder to capture the variance of the input.